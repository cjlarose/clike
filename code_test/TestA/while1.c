

void printint(int);
void printchar(char);

void main()
{
  int x, i;

  i = 10;
  while (i>=0) {
     printint(i);
     i = i - 2;
     printchar(10);  // newline
  };

}
  
