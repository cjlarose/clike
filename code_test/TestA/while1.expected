Loaded: /usr/local/share/spim/exceptions.s
10
8
6
4
2
0
